### PR TITLE
Spring Extensions 1.0.2 News Item and doc

### PR DESCRIPTION
**Pull request by Chessray**

### CODE REVIEW

File: content/documentation/reference-guide.md [hunk 1]
```diff
@@ -22,11 +27,6 @@ parent = "Documentation"
 * API documentation: [JavaDoc](/documentation/1.4/api/)
 * [Migration notes](https://github.com/mapstruct/mapstruct/releases/tag/1.4.2.Final)
 
-**Spring Extensions** (February 15th 2021; latest milestone release)
```
Comment (filiphr) [inline content/documentation/reference-guide.md:25]: I think that it is better to keep the extension release just below the last mapstruct core one

**Review by filiphr — CHANGES_REQUESTED** [review]
LGTM, maybe one small change for the location for the docs. See my comment

File: content/documentation/reference-guide.md (hunk 1 above)
Comment (Chessray) [reply content/documentation/reference-guide.md:25]: I figured you might say that. :laughing:

Comment (filiphr) [reply content/documentation/reference-guide.md:25]: I think that it can be a bit confusing otherwise. Thanks for the change. I merged this

**Pull request merged**